src/web/screens/AffiliationViews/DeploymentView/DetailsView/VersionView/components/DeployButton.tsx

```diff
@@ -47,14 +62,40 @@ export const DeployButton: React.FC<IDeployButtonProps> = ({
         minWidth="500px"
         maxWidth="800px"
       >
-        {releaseTo && <ReleaseToInformation releaseTo={releaseTo} />}
-        {children}
+        <div className={className}>
+          {isBranchDeleted && (
+            <MessageBar
+              type={MessageBar.Type.warning}
+              className="message-bar-width"
+            >
+              Applikasjonen har blitt deployet med en branch som er slettet.
```

Comment:
Synes at det bør spesifiseres at det er  **Aurora Config** branch som har blitt slettet. Ser det er gjort det for [MissingGitBranchInformation.tsx](https://github.com/Skatteetaten/aurora-konsoll/pull/177/files#diff-42c629fdd0e90f53e5a6ecd283c40b2eeb899b651bbb16ac4d99feedc78cb893)

Reply:
Godt poeng, skal få endret på det